src/components/item/index.tsx

```diff
@@ -42,6 +44,9 @@ const Product = ({
   authors,
   placeBid,
   disableShareDropdown,
+  imageHeight = 256,
```

Comment:
Is this needed? do every image needs this size? If not I would suggest to not parametrize the component

Reply:
The thumbnail images have all the same size.